lib/flipper/ui/actions/actors_gate.rb

```diff
@@ -24,7 +24,7 @@ def get
         def post
           feature_name = Rack::Utils.unescape(request.path.split('/')[-2])
           feature = flipper[feature_name.to_sym]
-          value = params["value"]
+          value = params["value"].strip
 
           if Util.blank?(value)
```

Comment:
looks like the code expects a possible `nil` value here, which means that your change on line `27` will raise an `undefined method for nil nilclass`

maybe `strip` inside the if block?